target/linux/ath79/dts/ar7161_mikrotik_routerboard-493ah.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar7100_mikrotik_routerboard-4xx.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-493ah", "qca,ar7161";
+	model = "MikroTik RouterBOARD 493AH";
+
+	keys: keys {
+		compatible = "gpio-keys";
+
+		status = "disabled";
```

Comment:
This doesn't make any sense.

Reply:
That's actually a bad cut and paste I put in an earlier comment suggesting this.

Yes, this line should be deleted, and the physical reset button should then be seen to work.